### DISCUSSION TITLE
Facing Webpack Compilation Error

### DISCUSSION THREAD

**Discussion by kousikmondal** [cypress-io/cypress#24504] [Questions and Help]
Getting a Web Compilation error, can someone help me to resolve this?

Error: Webpack Compilation Error
./cypress/support/commands.js
Module not found: Error: Can't resolve 'cypress/types/bluebird' in 'C:\Users\kousi\CypressfromScretch\cypress\support'
resolve 'cypress/types/bluebird' in 'C:\Users\kousi\CypressfromScretch\cypress\support'
  Parsed request is a module
  using description file: C:\Users\kousi\CypressfromScretch\package.json (relative path: ./cypress/support)
    Field 'browser' doesn't contain a valid alias configuration
    Looked for and couldn't find the file at the following paths:
[C:\Users\kousi\CypressfromScretch\cypress\support\node_modules]
[C:\Users\kousi\CypressfromScretch\cypress\node_modules]
[C:\Users\kousi\node_modules]
[C:\Users\node_modules]
[C:\node_modules]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird.js]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird.json]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird.jsx]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird.mjs]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird.coffee]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index.js]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index.json]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index.jsx]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index.mjs]
[C:\Users\kousi\CypressfromScretch\node_modules\cypress\types\bluebird\index.coffee]
 @ ./cypress/support/commands.js 32:4-37
 @ ./cypress/support/e2e.js

![Webpackissue](https://user-images.githubusercontent.com/79504741/199533280-8861ac69-0d27-425d-bb30-df617ad635d4.png)

**Comment by mamta-rani-25** [comment]
What is the solution for this error?

**Comment by al-mcnichol** [comment]
I'm just starting out with cypress and facing the same issue. Interestingly enough, I'm not seeing the same issue when I run the test locally
